sources/packages/backend/libs/sims-db/src/entities/education-program.model.ts

```diff
@@ -357,6 +358,14 @@ export class EducationProgram extends RecordDataModel {
   })
   effectiveEndDate?: string;
 
+  /**
+   * Returns `true` in case the effective end date has been reached or
+   * the effective end date is null. Otherwise `false`.
+   */
+  get isExpired(): boolean {
+    return isSameOrAfterDate(this.effectiveEndDate, new Date());
```

Comment:
Should we ensure the `new Date` is the date only using the `getISODateOnlyString`?
Maybe the `isSameOrAfterDate` can receive an option as `{ compareDateOnly: true }`.

Reply:
We don't need to make the current date be at 00h00min because whenever time it is, if it is greater or equal than the effectiveEndDate it'll be expired.